src/wirecloud/platform/wiring/utils.py

```diff
@@ -145,3 +108,307 @@ def get_wiring_skeleton() -> Wiring:
 def is_empty_wiring(visual_info: WiringVisualDescription) -> bool:
     return len(visual_info.connections) == 0 and len(visual_info.components.operator) == 0 and len(
         visual_info.components.widget) == 0
+
+
+def get_operator_cache_key(operator: 'CatalogueResource', domain: str, mode: str) -> str:
+    return f'_operator_xhtml/1/{domain}/{operator.id}?mode={mode}'
+
+
+async def get_operator_api_files(request: Request, theme: str) -> list[str]:
+    from src.wirecloud.platform.core.plugins import get_version_hash
+    key = f'operator_api_files/{get_current_domain(request)}?v={get_version_hash()}'
+    operator_api_files = await cache.get(key)
+
+    if operator_api_files is None or settings.DEBUG:
+        code = f'<script src="{get_absolute_static_url(f"js/main-{theme}-operator.js", request=request)}" />'
+
+        doc = etree.parse(BytesIO(('<files>' + code + '</files>').encode('utf-8')), etree.XMLParser())
+
+        files = [script.get('src') for script in doc.getroot()]
+        operator_api_files = tuple([get_absolute_static_url(file, request=request, versioned=True) for file in files])
```

Comment:
Con esto y eliminando lo de code, doc y files es suficiente:

```suggestion
        operator_api_files = tuple([get_absolute_static_url(f"js/main-{theme}-operator.js", request=request, versioned=True)])
```